VirtoCommerce.CartModule.Data/Model/CouponEntityComparer.cs

```diff
@@ -0,0 +1,33 @@
+using System.Collections.Generic;
+
+namespace VirtoCommerce.CartModule.Data.Model
+{
+    public class CouponEntityComparer : IEqualityComparer<CouponEntity>
```

Comment:
This Comparer is redundant need to use AnonymousComparer instead  https://github.com/VirtoCommerce/vc-module-catalog/blob/master/VirtoCommerce.CatalogModule.Data/Model/CatalogEntity.cs#L113